### PR TITLE
Usable items

### PR DESCRIPTION
**Pull request by KarolDenst**
This pr makes the following changes:
- To dig select hammer and press left mouse button
- To mine select hammer and press right mouse button
- To launch projectile select gun and press left mouse button (gun won't fire if there are no bullets in the inventory)
- Mining now affects all chunks that are close enough (not just one)

### CODE REVIEW

File: Hyper/Controllers/ProjectilesController.cs [hunk 1]
```diff
@@ -68,7 +67,6 @@ public void Render()
     public void RegisterCallbacks(Context context)
     {
         context.RegisterUpdateFrameCallback((e) => UpdateProjectiles((float)e.Time));
-        context.RegisterKeyDownCallback(Keys.P, CreateProjectile);
```
Comment (AleksyBalazinski) [inline Hyper/Controllers/ProjectilesController.cs:71]: `CreateProjectile` is now unused

File: Hyper/PlayerData/InventorySystem/Items/Gun.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    
+    public override bool IsStackable => false;
+    
+    public override void Use(Scene scene)
+    {
+        if (!scene.Player.Inventory.TryRemoveItem("bullet")) return;
+        CreateProjectile(scene);
+    }
+    
+    private void CreateProjectile(Scene scene)
```
Comment (AleksyBalazinski) [inline Hyper/PlayerData/InventorySystem/Items/Gun.cs:19]: can be static

File: Hyper/PlayerData/Player.cs [hunk 3]
```diff
@@ -4,35 +4,35 @@
...
     public System.Numerics.Vector3 RayOrigin => PhysicalCharacter.Pose.Position
-            + Conversions.ToNumericsVector(ViewDirection) * _rayOffset;
+            + Conversions.ToNumericsVector(ViewDirection) * RayOffset;
 
     public int RayId => 0;
 
     public Player(PhysicalCharacter physicalCharacter, Context context, int currentSphereId = 0) : base(physicalCharacter, currentSphereId)
     {
-        Inventory = new Inventory(context);
+        Inventory = new Inventory(context, true);
```
Comment (AleksyBalazinski) [inline Hyper/PlayerData/Player.cs:35]: can we use named arguments whenever a literal is passed to a method? It's easier to read

File: Hyper/Scene.cs [hunk 4]
```diff
@@ -115,7 +131,7 @@ public void Dispose()
     {
         foreach (var chunk in Chunks)
             chunk.Dispose(SimulationManager.Simulation, SimulationManager.BufferPool);
-
+
```
Comment (AleksyBalazinski) [inline Hyper/Scene.cs:134]: some white char

**Review by AleksyBalazinski — CHANGES_REQUESTED** [review]
Some minor comments

File: Hyper/PlayerData/Player.cs (hunk 3 above)
Comment (KarolDenst) [reply Hyper/PlayerData/Player.cs:35]: I might forget about this because resharper does it for me but I'll try

**Review by AleksyBalazinski — COMMENTED** [review]
I noticed that there's a problem with mining on chunk edges in spherical 
image
The error was probably introduced in the previous commit on main. Can we try to fix it in this PR?

**Review by AleksyBalazinski — APPROVED** [review]

**Pull request merged**